README.md

```diff
@@ -345,43 +351,43 @@ There are two ways to achieve this (both involve using a CSS variable):
 
 1. Declare a CSS variable directly in the styles and set its value using inline styles:
 
-```jsx
-const Heading = styled('h1')({
-  color: 'var(--color)',
-});
+   ```jsx
```

Comment:
Is the code blocks indentation expected? Other code blocks here aren't.

Reply:
Yes, it's expected. These code blocks are nested inside list items, so they should be indented. Other code blocks are not indented because they're not inside list items (unless I missed any).

|before|after|
|---|---|
|<img width="958" alt="Screenshot 2024-10-09 at 20 47 03" src="https://github.com/user-attachments/assets/ff39d6ad-f161-4f79-a84a-7082f4fea38a">|<img width="956" alt="Screenshot 2024-10-09 at 20 44 50" src="https://github.com/user-attachments/assets/8ce279c3-d338-4091-aa0a-bfd926b8b1c6">|